storage/persistent_storage/persistent-storage-ocs.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 {rh-storage-first} is a provider of agnostic persistent storage for {product-title} supporting file, block, and object storage, either in-house or in hybrid clouds. As a Red Hat storage solution, {rh-storage-first} is completely integrated with {product-title} for deployment, management, and monitoring.
 
-{rh-storage-first} provides its own documentation library. The complete set of {rh-storage-first} documentation identified below is available at https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9.
+{rh-storage-first} provides its own documentation library. The complete set of {rh-storage-first} documentation identified below is available at https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10.
```

Comment:
I know this is just an update of existing text, but links should have friendly text. Suggestion:
```suggestion
{rh-storage-first} provides its own documentation library. The complete set of {rh-storage-first} documentation identified below is available in the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10[Product Documentation for Red Hat OpenShift Data Foundation 4.10].
```

Reply:
Addressed